diceware/__init__.py

```diff
@@ -208,4 +210,12 @@ def main(args=None):
     if options.version:
         print_version()
         raise SystemExit(0)
-    print(get_passphrase(options))
+    try:
+        print(get_passphrase(options))
+    except (OSError, IOError) as infile_error:
+        if getattr(infile_error, 'errno', 0) == ENOENT:
+            logging.getLogger('ulif.diceware').error(
+                "The file '%s' does not exist." % infile_error.filename)
+            raise SystemExit(1)
+        else:
+            raise
```

Comment:
The else branch is not covered in tests. This is revealed by coverage tests.

You can test all this on your commandline when running `tox`.

 `$ tox -e coverage`

for coverage only, or just

`$ tox`

for running tests in all envs (including coverage and flake tests). If these tests pass, normally also travis will be happy.

Reply:
Thank you, I'm using `tox` now.

Reply:
In a test, try to provoke some IOError, OSError, which is not a file-not-found condition. Maybe you can request to use 'mylist' which you created as a directory before? Any kind of file-reading problem except file-not-found should do.